lib/bundle_update_interactive/outdated_gem.rb

```diff
@@ -2,19 +2,21 @@
 
 module BundleUpdateInteractive
   class OutdatedGem
-    attr_reader :current_lockfile_entry, :updated_lockfile_entry, :gemfile_groups
-    attr_writer :vulnerable
+    attr_accessor :name,
```

Comment:
🗒️ This refactor removes dependencies on lock file entries (which are deep data structures that in turn depend on internal Bundler classes) in favor of a flat set of simple PORO attributes.